src/transaction/transaction.rs

```diff
@@ -895,10 +901,16 @@ impl TransactionOptions {
     }
 }
 
-/// The default TTL of a lock in milliseconds.
+const MANAGED_TTL: u64 = 20000;
+/// The default TTL of a lock in milliseconds
 const DEFAULT_LOCK_TTL: u64 = 3000;
-/// The default heartbeat interval.
-const DEFAULT_HEARTBEAT_INTERVAL: Duration = Duration::from_secs(1);
+/// The default heartbeat interval in milliseconds
+const DEFAULT_HEARTBEAT_INTERVAL: Duration = Duration::from_secs(MANAGED_TTL / 2);
+/// TiKV recommends each RPC packet should be less than ~1MB. We keep each packet's
+/// Key+Value size below 16KB.
```

Comment:
I don't understand this comment, why does packet <1mb => K+V <16KB ?

Reply:
This comment is copied from the tikv client of TiDB.

Reply:
Could you just link to that comment rather than copying it please?